src/js/components/Text/StyledText.js

```diff
@@ -46,6 +46,14 @@ const wordBreakStyle = css`
   word-break: ${props => props.wordBreak};
 `;
 
+const fontFamily = props => {
+  return props.theme.text.font
+    ? css`
+        font-family: ${props.theme.text.font.family};
+      `
+    : '';
```

Comment:
This empty string will change our default behavior. 
We should default to the existing font family in case the font object doesn't exist instead of setting in it a new default.

Reply:
I am still seeing the same snapshot changes when I remove all behavior from
`const fontFamil = props => {` (function at line 49 of StyledText.js). I don't think the snapshot changes are happening because of this. I think that the snapshot class name changes when `$props => fontFamily(props)` is added at line 78. So I think the snapshot changes are unavoidable.

As far as changing the default behavior goes I modeled this code off of what was already done in StyledHeading.js. I don't think this code is changing the default behavior. If `props.theme.text.font` is not specified the default font is still showing up.

I could change the function to be:
```javascript
const fontFamily = props => {
    return props.theme.text.font;
}   
...
${props => {if(props.theme.text.font.family){fontFamily(props)}}}
```

Reply:
I think something close to
```
${props => { props.theme.text.font && props.theme.text.font.family && fontFamily(props) }
```
Would be cleanest

Reply:
Changed to:

```javascript
const fontFamily = css`
  font-family: ${props => props.theme.text.font.family};
`;
...
${props =>
    props.theme.text.font &&
    props.theme.text.font.family &&
    fontFamily}
```